src/gui/guiFormSpecMenu.cpp

```diff
@@ -3172,6 +3165,14 @@ void GUIFormSpecMenu::regenerateGui(v2u32 screensize)
 	gui::IGUIFont *old_font = skin->getFont();
 	skin->setFont(m_font);
 
+	// Add a new element that will hold all the background elements as its children
+	// because it is the first added element, all backgrounds will be behind all
+	// the other elements
```

Comment:
```suggestion
	// Add a new element that will hold all the background elements as its children.
	// Because it is the first added element, all backgrounds will be behind all
	// the other elements.
```